tests/Common/AbstractMigrationBuilderTest.php

```diff
@@ -41,51 +39,54 @@ public function testExecute(): void
 
     public function testInsert(): void
     {
-        $this->createTable('test', ['id' => 'int']);
-
+        $this->builder->createTable('test', ['id' => $this->builder->integer()]);
         $this->builder->insert('test', ['id' => 1]);
 
-        $this->assertSame(
+        $this->assertEquals(
             '1',
-            (string) $this->db->createCommand('SELECT count(*) FROM {{test}} WHERE [[id]] = 1')->queryScalar()
+            $this->db->createCommand('SELECT count(*) FROM {{test}} WHERE [[id]] = 1')->queryScalar()
         );
         $this->assertInformerOutputContains('    > Insert into test ... Done in ');
+
+        $this->builder->dropTable('test');
```

Comment:
Is this really necessary? Tables removed in `tearDown`.

Reply:
I forgot to delete it, since i delete it in `tearDown()` it is correct.

Reply:
done.